dockerbot.py

```diff
@@ -4,10 +4,32 @@
 import subprocess, os, sys
 from telepot.loop import MessageLoop
 
+#Which Signing Website
+v_SIGN_PARENTS_EDUCATION_GOV_IL = int(os.getenv('SIGN_WEBSITE_EDUCATION_GOV_IL'))
+v_SIGN_WEBSITE_MASHOV = int(os.getenv('SIGN_WEBSITE_MASHOV'))
```

Comment:
why is it neededd? 
once you chose the command in telegram you get the script with the URL inside or is there another reason?